apiserver/spaces/spaces.go

```diff
@@ -102,3 +104,19 @@ func (api *spacesAPI) ListSpaces() (results params.ListSpacesResults, err error)
 	}
 	return results, nil
 }
+
+// RefreshSpaces refreshes spaces from substrate
+func (api *spacesAPI) ReloadSpaces() error {
+	canRead, err := api.authorizer.HasPermission(permission.ReadAccess, api.backing.ModelTag())
+	if err != nil && !errors.IsNotFound(err) {
+		return errors.Trace(err)
+	}
+	if !canRead {
+		return common.ServerError(common.ErrPerm)
+	}
+	env, err := environs.GetEnviron(api.backing, environs.New)
```

Comment:
this looks a little odd to use api.backing to get the environ, in order to pass the environ into api.backing.ReloadSpaces.
Is there a reason api.backing.ReloadSpaces doesn't grab its own environ?

This may be better for layering/testing, it just looks a little odd.